test/dynamo/test_subclasses.py

```diff
@@ -312,19 +312,20 @@ def forward(self, L_x_ : torch.Tensor):
 
         wrap_body_0 = self.wrap_body_0
         wrap = torch._higher_order_ops.wrap.wrap(wrap_body_0, l_x_);  wrap_body_0 = l_x_ = None
-        return (wrap,)
+        getitem = wrap[0];  wrap = None
+        return (getitem,)
 
     class GraphModule(torch.nn.Module):
         def forward(self, l_x_):
             add_ = l_x_.add_(1.0);  l_x_ = None
-            return add_
+            return (add_,)
 """
-        check_count_and_graph(1, 1, 1, expected_graph)
+        check_count_and_graph(1, 2, 1, expected_graph)
 
         ff = torch.func.functionalize(f)
         ff_out = ff(t_clone)
         # frame count and op count are incremented due to re-compilation
-        check_count_and_graph(2, 2, 2, expected_graph)
+        check_count_and_graph(2, 4, 2, expected_graph)
```

Comment:
What is the middle number? Why did it go up?

Reply:
It's the number of expected operations. What's happening there is:
- `wrap` returns a tuple (consistent with plain dynamo)
- A new node `call_function(get_item)` is needed to create a proxy for the inner `TensorVariable` (+1 extra op)
So, we are doubling the number of expected operations if `wrap` returns a single tensor.